environments/go/builder/build.sh

```diff
@@ -22,6 +22,15 @@ fi
 
 cd ${srcDir}
 
+# use private vendor is the vendor dir is exsit
+if  [ -d "vendor" ] && [ ! -z ${GOLANG_VERSION} ] && version_ge ${GOLANG_VERSION} "1.11"; then
```

Comment:
Fission enables GO111MODULE environment variable after 1.12, so here we can set greater and equal to 1.12
```
version_ge ${GOLANG_VERSION} "1.12"
```